cfme/tests/configure/test_analysis_profiles.py

```diff
@@ -70,6 +71,7 @@ def analysis_profile_collection(appliance):
 
 @pytest.mark.sauce
 @pytest.mark.tier(2)
+@pytest.mark.meta(blockers=[BZ(1727929, forced_streams='5.11')])
```

Comment:
I dont think you need `forced_streams`

```
In [1]: from cfme.utils.blockers import BZ

In [2]: BZ(1727929).blocks
Out[2]: True

In [4]: app.version
Out[4]: Version('5.11.0.13')
```

Reply:
I added forced_streams because the BZ was originally created with a Target Release of 'cfme-future', which doesn't match against the logic in BZ.blocks(). I've removed it.